python/ray/experimental/workflow/workflow_access.py

```diff
@@ -193,16 +200,38 @@ def report_success(self, workflow_id: str) -> None:
         self._workflow_outputs.pop(workflow_id, None)
 
 
+def init_management_actor() -> None:
+    """Initialize WorkflowManagementActor"""
+    store = storage.get_global_storage()
+    try:
+        workflow_manager = ray.get_actor(MANAGEMENT_ACTOR_NAME)
+        is_valid = workflow_manager.validate_storage.remote(store.storage_url)
```

Comment:
I think here it's wrong, right? You need ray.get to get the internal value. Btw, this logic should belong to init_management_actor, not the actual actor.

How about this
```
if ray.get(workflow_manager.storage_url.remote()) != storage_url:
    raise RuntimeError(...)
```